js/core/jquery-enhanced.js

```diff
@@ -0,0 +1,173 @@
+define(["jquery","core/utils"], function($, utils) {
```

Comment:
This is a straight copy and paste from core/utils (where this code used to live). This code does not require review.